drivers/usb/device/usb_dc_sam0.c

```diff
@@ -21,8 +21,7 @@ LOG_MODULE_REGISTER(usb_dc_sam0);
 
 #define USB_SAM0_IN_EP 0x80
 
-#define REGS ((Usb *)DT_USB_DC_SAM0_BASE_ADDRESS)
```

Comment:
Please keep using the base address from dts rather than from ASF. We are doing this by convention for all the vendors

Reply:
ACK, changed.